lib/mquery.js

```diff
@@ -1713,6 +1751,89 @@ Query.prototype.tailable = function() {
   return this;
 };
 
+/**
+ * Sets the specified number of `mongod` servers, or tag set of `mongod` servers,
+ * that must acknowledge this write before this write is considered successful.
+ * This option is only valid for operations that write to the database:
+ *
+ * - `deleteOne()`
+ * - `deleteMany()`
+ * - `findOneAndDelete()`
+ * - `findOneAndUpdate()`
+ * - `remove()`
+ * - `update()`
+ * - `updateOne()`
+ * - `updateMany()`
+ *
+ * Defaults to the `w` value if it is specified in writeConcern options
+ *
+ * ####Example:
+ *
+ * mquery().writeConcern(0)
+ * mquery().writeConcern(1)
+ * mquery().writeConcern({ w: 1, j: true, wtimeout: 2000 })
+ * mquery().writeConcern('majority')
+ * mquery().writeConcern('m') // same as majority
+ * mquery().writeConcern('tagSetName') // if the tag set is 'm', use .writeConcern({ w: 'm' }) instead
+ * mquery().w(1) // w is alias of writeConcern
+ *
+ * @method writeConcern
+ * @memberOf Query
+ * @instance
+ * @param {String|number|object} concern 0 for fire-and-forget, 1 for acknowledged by one server, 'majority' for majority of the replica set, or [any of the more advanced options](https://docs.mongodb.com/manual/reference/write-concern/#w-option).
+ * @see mongodb https://docs.mongodb.com/manual/reference/write-concern/#w-option
+ * @return {Query} this
+ * @api public
+ */
+
+Query.prototype.writeConcern = Query.prototype.w = function writeConcern(concern) {
+  if ('object' === typeof concern) {
+    if ('undefined' !== typeof concern.j) this.options.j = concern.j;
+    if ('undefined' !== typeof concern.w) this.options.w = concern.w;
+    if ('undefined' !== typeof concern.wtimeout) this.options.wtimeout = concern.wtimeout;
+  } else {
+    this.options.w = 'm' === concern ? 'majority' : concern;
```

Comment:
I'm not sure I like supporting 'm' as a shortcut for 'majority'. Is there some precedent for this?

Reply:
mquery ReadPreference `.read()` support it `m` = `majority`
https://github.com/aheckmann/mquery/blob/45c4bd104613da47828df13d4aec12c324525c53/lib/mquery.js#L1583-L1596

So in my previous request, I make `readConcern()` support it. `m` = `majority`
https://github.com/aheckmann/mquery/blob/45c4bd104613da47828df13d4aec12c324525c53/lib/mquery.js#L1640-L1653

So I make `writeConcern()` to accept 'm' as shortcut as well.
I doubt if it is a good to support `m` in write concern too as there maybe conflict if the tag name happens to be `m`

For consistency, maybe we should make a breaking change that readConcern doesn't accept `m` either? Only read preference accept alias names.

In read concern, `l`=`local`, `lz` = `linearizable`. I think `lz` is not good abbreviation but I can't find good alternative.

Reply:
Yeah you're right, this is more consistent with the readConcern and read APIs, let's merge this